CalibMuon/RPCCalibration/python/l1RPC_EventContent_cff.py

```diff
@@ -2,7 +2,8 @@
 
 l1RPC_EventContent = cms.PSet(
     outputCommands = cms.untracked.vstring('drop *', 
-        'keep RPCDetIdRPCDigiMuonDigiCollection_*_*_*', 
+        #'keep RPCDetIdRPCDigiMuonDigiCollection_*_*_*', 
+        'keep RPCDetIdRPCDigiMuonDigiCollection_muonRPCDigis_*_*', 
```

Comment:
I didn't find this file included in standard workflows with RECO step.
We have other digi producers (simMuon\* ; hltMuon\*).

So, changes in this file should probably not be made.